### PR TITLE
SGC-2079 : [Coddestyle] Add readme for Java and JS

### PR DESCRIPTION
**Pull request by Serykh92**
### Why?

There was a need for a description and instructions

### What's Changing?

This PR adds a description of the project and instructions for adding the "com.kaseya.static-analyzers" package to the user's project. I also added a file that describes the contribution rules

### CODE REVIEW

File: .github/workflows/Contributors.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Pull requests must be merged only from Kaseya developers
```
Comment (andrey-kozel) [inline .github/workflows/Contributors.md:1]: by Kaseya developers

**Review by andrey-kozel — DISMISSED** [review]

File: .github/workflows/Contributors.md (hunk 1 above)
Comment (aglumova) [inline .github/workflows/Contributors.md:1]: Please change name to `CONTRIBUTORS.md` and place to the `.github`

File: README.md [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
-2. < Github actions and Github packages >
-3. < Ho to download Github package >
-4. ./github/CONTRIBUTING.md ----> <Pull requests must be merged only from Kaseya developers>
-5. < Add README.md for Java folder > -> < How to download it from github packages in Maven(pom.xml) and Gradle (build.gradle) >   
-6. < Update java/.mvn/wrapper > -> < maven local > -> < mvn -N wrapper:wrapper >
-7. < Fixed Ruby actions >
+This repository provides a set of code styles for several programming languages: [Ruby](https://github.com/SpanningCloudApps/codestyle/blob/main/ruby/), [Java](https://github.com/SpanningCloudApps/codestyle/tree/main/java), [C#](https://github.com/SpanningCloudApps/codestyle/blob/main/csharp/) and [JavaScript](https://github.com/SpanningCloudApps/codestyle/blob/main/js/)
+
+
+
```
Comment (aglumova) [inline README.md:6]: Please remove a lot of new lines, please use 1 empty line as separator.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,22 @@
...
+## Github actions and Github packages
+   We use Github actions to create a projects assembly and publish packages to Github. Files with instructions here:
+   [C#](https://github.com/SpanningCloudApps/codestyle/blob/main/.github/workflows/deploy-charp.yml),
+   [Java](https://github.com/SpanningCloudApps/codestyle/blob/main/.github/workflows/deploy-java.yml),
+   [JavaScript](https://github.com/SpanningCloudApps/codestyle/blob/main/.github/workflows/deploy-js.yml),
+   [Ruby](https://github.com/SpanningCloudApps/codestyle/blob/main/.github/workflows/deploy-ruby.yml).
+
+   Ready-made packages [here](https://github.com/orgs/SpanningCloudApps/packages?repo_name=codestyle)
```
Comment (aglumova) [inline README.md:15]: For what is it?

File: java/Readme.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
+## How to download package from github packages and configure it in Maven and Gradle
```
Comment (aglumova) [inline java/Readme.md:1]: Please change file nae according to the similar naming for existing files.

File: java/Readme.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+Very important: you must have an account on [github](https://github.com/)
+
+1. ### Create access token
+Open your github account and click on your icon in the upper right part of the screen and select "Settings" as on the screen below  
+![](../../../Снимок экрана 2022-08-30 в 12.10.41.png)
```
Comment (aglumova) [inline java/Readme.md:7]: - Please change name to the English language
- Push screenshots to the docs/images folder and pay attention that docs must to be in the target after compilation.

File: java/Readme.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+```
+mvn install
+```
+For Gradle
+```
+gradlew build
+```
+Or you can just update project in your IDE for both options
+
+5. ### For Maven Wrapper users
```
Comment (aglumova) [inline java/Readme.md:82]: Please remove it, for what it?

File: .github/workflows/Contributors.md (hunk 1 above)
Comment (Serykh92) [reply .github/workflows/Contributors.md]: fixed

File: README.md (hunk 2 above)
Comment (Serykh92) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (Serykh92) [reply README.md]: deleted

File: java/Readme.md (hunk 5 above)
Comment (Serykh92) [reply java/Readme.md]: fixed

File: java/Readme.md (hunk 6 above)
Comment (Serykh92) [reply java/Readme.md]: fixed

File: java/Readme.md (hunk 4 above)
Comment (Serykh92) [reply java/Readme.md]: fixed

**Pull request closed without merging**